docs/chapters/workflow/release/howto-release-pelux.rst

```diff
@@ -0,0 +1,98 @@
+Creating a PELUX release
+========================
```

Comment:
Would it make sense to separate the source release from a more full release? Or would that be just a matter of how far one follows a check-list eventually sort of?

Reply:
As far as I can tell so far, making a binary release (I assume that's what you mean by full) means you have to do the same steps plus some more. That is, all steps for a source release are relevant for a binary release as well.

I think, when we do a binary release, that we can append to these steps with conditional steps for a binary release.

Reply:
Yeah I see what you mean. For now I think it's a fair assumption. I can imagine that a typically humongous (relatively) automotive release might have steps both before, in the middle, and after these steps but that's theoretical at the moment. I'm OK with this.